lib/commands/exam.js

```diff
@@ -101,6 +116,47 @@ module.exports = TestCommand.extend({
     return this._super.run.apply(this, arguments);
   },
 
+  /**
+   *
+   * Instantiate an executionMappingObj object
+   *
+   * @param {Object} option
+   */
+  getExecutionMappigInstance: function(options) {
+    if (executionMappingObj !== undefined && executionMappingObj['numberOfBrowsers'] !== undefined && executionMappingObj['executionMapping'] !== undefined) {
+      return executionMappingObj;
+    }
+    const filePath = options.replayExecution;
+    const replayBrowser = options.replayBrowser;
+
+    let testMappingObj = {};
+
+    try {
+      // Read the replay execution json file.
+      const testMappingRawData = fs.readFileSync(filePath);
+      testMappingObj = JSON.parse(testMappingRawData);
+    } catch (err) {
+      const errorMsg = 'Error reading reply execution JSON file - ' + err;
+      throw new Error(errorMsg);
+    }
+
+    const browserModuleMappingObj = testMappingObj['executionMapping'];
+
+    executionMappingObj['numberOfBrowsers'] = testMappingObj['numberOfBrowsers'];
+    executionMappingObj['executionMapping'] = ((replayBrowser, browserModuleMappingObj) => {
+      const browserModuleMap = {};
+
+      for (const idx in replayBrowser) {
```

Comment:
`for (const browserId in browserIdsToReplay) {`, if you make the suggested renaming above.